spec/features/info_spec.rb

```diff
@@ -1,11 +1,41 @@
 require "rails_helper"
 require "govuk/client/test_helpers/metadata_api"
+require "gds_api/test_helpers/content_store"
 
 feature "Info page" do
   include GOVUK::Client::TestHelpers::MetadataAPI
+  include GdsApi::TestHelpers::ContentStore
+
+  before do
+    content_examples = GovukContentSchemaTestHelpers::Examples.new
+    example_need = JSON.parse(content_examples.get("need", "need"))
+    example_content = JSON.parse(content_examples.get("specialist_document", "aaib-reports"))
+
+    @apply_for_a_uk_visa_need = example_need.merge(
```

Comment:
It may be a good idea to use the `govuk_schemas` to generate these payloads. That will prevent them from becoming invalid if the `aaib_report` or `need` schemas ever change. It will also make you less reliant on the examples, which could be changed from under you.

https://github.com/alphagov/govuk_schemas_gem/blob/master/lib/govuk_schemas/random_example.rb#L56
For examples:

- https://github.com/alphagov/publishing-api/blob/93e7d0ac5ba5159416e4ffbb9f242a03b3562f41/spec/integration/message_queue_publishing_spec.rb#L43-L63
- https://github.com/alphagov/govuk_navigation_helpers/blob/e1e99cdc8cf734470234c3c2655d7176f49d05ad/spec/breadcrumbs_spec.rb#L90-L91